kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -58,6 +58,7 @@ Export-Package: org.eclipse.kura;version="1.7.0",
  org.eclipse.kura.net.modem;version="2.4.0",
  org.eclipse.kura.net.route;version="1.0.2",
  org.eclipse.kura.net.wifi;version="2.3.0",
+ org.eclipse.kura.net2;version="1.0.0",
```

Comment:
To avoid boxing us into a corner, can we signal this as unstable for the first release?

```suggestion
 org.eclipse.kura.net2;version="0.1.0",
```

Reply:
Indeed, we'd discuss about this. Let's leave 1.0.0 for now.